lib/postgres.js

```diff
@@ -5,11 +5,11 @@
  * ==================================================
  * Postgresql client
  * ==================================================
- * 
+ *
  * Provides access to the geo database
  */
 
-var pg = require('pg').native;
+var pg = require('pg');
```

Comment:
There appears to be a bug in `node-pg` or `node-pg-native` that breaks the row accumulator: https://github.com/brianc/node-postgres/issues/953